src/scene/graphics/shared/utils/__tests__/toFillStyle.test.ts

```diff
@@ -0,0 +1,139 @@
+import { Color } from '../../../../../color/Color';
+import { Texture } from '../../../../../rendering/renderers/shared/texture/Texture';
```

Comment:
Recommend using the src alias `~`
```suggestion
import { Color } from '~/color/Color';
import { Texture } from '~/rendering/renderers/shared/texture/Texture';
```